(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> cannot concatenate 'str' and 'NoneType' objects"
p3
sS'layer'
p4
S'/home/joseph/BabbleBot/applications/BabbleBot/controllers/core.py'
p5
sS'code'
p6
S'<code object <module> at 0x7fc304553230, file "/home/joseph/BabbleBot/applications/BabbleBot/controllers/core.py", line 2>'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/joseph/BabbleBot/gluon/restricted.py", line 219, in restricted\n    exec(ccode, environment)\n  File "/home/joseph/BabbleBot/applications/BabbleBot/controllers/core.py", line 73, in <module>\n  File "/home/joseph/BabbleBot/gluon/globals.py", line 419, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/joseph/BabbleBot/applications/BabbleBot/controllers/core.py", line 18, in guestCheck\n    auth = groupMe.checkValidAuth(personAuth)\n  File "applications/BabbleBot/modules/groupMe.py", line 17, in checkValidAuth\n    output = requests.get("https://api.groupme.com/v3/groups?token=" + auth)\nTypeError: cannot concatenate \'str\' and \'NoneType\' objects\n'
p11
s.